docs/plugins/metasync.rst

```diff
@@ -29,10 +31,23 @@ Configuration
 To configure the plugin, make a ``metasync:`` section in your configuration
 file. The available options are:
 
-- **source**: A list of sources to fetch metadata from. Set this to "amarok"
-  to enable synchronization with that player.
+- **source**: A list of comma-separated sources to fetch metadata from.
+  Set this to "amarok" or "itunes" to enable synchronization with that player.
   Default: empty
 
+The follow subsections describe additional configure required for some players.
+
+itunes
+''''''
+
+The path to your iTunes library **xml** file has to be configured, e.g.::
+
+    metasync:
+        source: itunes
+        itunes:
+            library: ~/Music/iTunes Library.xml
```

Comment:
Shall we provide a default location for the library XML? It seems like most users will probably have theirs here (at least on OS X).